src/Humanizer/Configuration/FormatterRegistry.cs

```diff
@@ -2,8 +2,27 @@
 {
     class FormatterRegistry : LocaliserRegistry<IFormatter>
     {
-        public FormatterRegistry() : base(new DefaultFormatter("en-US"))
+        public static FormatterRegistry CreateInstance()
         {
+            try
+            {
+                return new FormatterRegistry(new DefaultFormatter("en-US"), false);
```

Comment:
This try-catch does not guarantee that exception is expected at exactly `new DefaultFormatter("en-US")` and in particular at `_culture = new(localeCode)` part. It can be thrown anywhere inside the `FormatterRegistry` constructor which creates many more cultures.

I think it need to be localized to something like this:

```csharp
CultureInfo GetDefaultCulture() {
  try {
    return new CultureInfo("en-US");
  } catch (CultureNotFoundException) {
    return CultureInfo.InvariantCulture;
  }
}
```
And then check if default culture is invariant and if so then skip all other initializers.